library/Icingadb/Common/BaseTableRowItem.php

```diff
@@ -79,6 +81,20 @@ protected function createVisual()
         return $visual->isEmpty() ? null : $visual;
     }
 
+    protected function setMultiselectFilter(Rule $filter)
+    {
+        $this->addAttributes(['data-icinga-multiselect-filter' => '(' . QueryString::render($filter) . ')']);
+
+        return $this;
+    }
+
+    protected function setDetailFilter(Rule $filter)
+    {
+        $this->addAttributes(['data-icinga-detail-filter' => '(' . QueryString::render($filter) . ')']);
+
+        return $this;
+    }
```

Comment:
The list already provides this with `$this->list->addDetailFilterAttribute()` etc.